src/Components/LocationModal.tsx

```diff
@@ -63,167 +181,159 @@ const LocationModal = ({ location, onClose }: ModalProps) => {
         type: 'boolean',
         title: labelMap[key].card,
         key,
-        icon: <InfoIcon />,
+        icon: labelMap[key].icon,
       });
     });
 
+  const address = `${location.location_address_street.trim()}, ${location.location_address_locality.trim()}, ${location.location_address_region.trim()} ${location.location_address_postal_code.trim()}`;
+
   return (
     <Modal
-      style={{
-        width: '90%',
-        height: '95%',
-        overflow: 'auto',
-        maxWidth: 600,
-        padding: 10,
-        margin: '0 auto',
-        outline: 0,
-      }}
+      className={classes.modal}
       onClose={() => onClose()}
       disableAutoFocus
       open
     >
-      <Card style={{ outline: 0 }}>
+      <Card className={classes.card}>
         <CardHeader
           title={location.location_name}
-          subheader={`${location.location_address_street.trim()}, ${location.location_address_locality.trim()}, ${location.location_address_region.trim()} ${location.location_address_postal_code.trim()}`}
+          subheader={address}
+          className={classes.cardHeader}
           action={
-            <IconButton aria-label="close" onClick={() => onClose()}>
-              <CloseIcon />
-            </IconButton>
+            <div>
+              <IconButton
+                area-label="call"
+                href={`tel://${location.location_contact_phone_main}`}
+              >
+                <PhoneIcon />
+              </IconButton>
+              <IconButton
+                area-label="directions"
+                href={`https://maps.google.com/?&daddr=${address}`}
+                target="_blank"
```

Comment:
Watch out for this https://developers.google.com/web/tools/lighthouse/audits/noopener